geode-core/src/main/java/org/apache/geode/internal/cache/DestroyPartitionedRegionMessage.java

```diff
@@ -111,9 +113,26 @@ public static DestroyPartitionedRegionResponse send(Set recipients, PartitionedR
     return resp;
   }
 
+  @Override
+  protected Throwable processCheckForPR(PartitionedRegion pr,
+      DistributionManager distributionManager) {
+    if (pr != null && !pr.getDistributionAdvisor().isInitialized()) {
+      Throwable thr = new ForceReattemptException(
+          LocalizedStrings.PartitionMessage_0_COULD_NOT_FIND_PARTITIONED_REGION_WITH_ID_1
+              .toLocalizedString(distributionManager.getDistributionManagerId(),
+                  pr.getRegionIdentifier()));
+      return thr; // reply sent in finally block below
```

Comment:
is //reply sent in finally block below still valid?

Same comment is in the other implementation of processCheckForPR